### PR TITLE
Product listing refactor

### PR DESCRIPTION
**Pull request by kart-c**
## Product Listing refactor

- Filter section is sticky 
- Changed card layout 
- changed grid layout

### Before 

#

### CODE REVIEW

**Review by RahullRawat — APPROVED** [review]

File: pages/product-listing/product-listing.css [hunk 1]
```diff
@@ -85,12 +117,46 @@
 	background-color: var(--primary-color);
```
Comment (Ayushi20-19) [inline pages/product-listing/product-listing.css:117]: background-color applied twice

Comment (Ayushi20-19) [inline pages/product-listing/product-listing.css:117]: background-color applied twice

**Review by Ayushi20-19 — APPROVED** [review]

**Review by RakshaPawar108 — APPROVED** [review]
Looks good to merge Kartikeya!

File: pages/product-listing/product-listing.css (hunk 1 above)
Comment (kart-c) [reply pages/product-listing/product-listing.css:117]: # 
Hey Ayushi, the code is collapsed here, I think that might have confused you to think it's repeating twice. If you expand the code it is not repeated.

**Pull request merged**